api/v1/app.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/python3
+from flask import Flask, jsonify
+from models import storage
+from api.v1.views import app_views
+import os
+
+app = Flask(__name__)
+app.register_blueprint(app_views, url_prefix="/api/v1")
+app.url_map.strict_slashes = False
+
+@app.teardown_appcontext
+def close(err):
+    storage.close()
+
+@app.errorhandler(404)
+def page_not_found_404(e):
+    '''
+        This will return 404 not found error.
+    '''
+    return jsonify({"error": "Not found"}), 404
+
+
+if __name__ == "__main__":
+    host=os.getenv("HBNB_API_HOST")
```

Comment:
_pushes up glasses_
You could have used the default, argument for getenv to use "0.0.0.0" in the event of the variable not being set.

Reply:
Interesting, I didn't know there was a default value for getenv. What do you mean? Example?

Reply:
`host = os.getenv("HBNB_API_HOST", "0.0.0.0")`
If no value for the variable `HBNB_API_HOST` is found, getenv will default to `0.0.0.0`